homeassistant/components/tesla/__init__.py

```diff
@@ -147,12 +169,24 @@ async def async_setup_entry(hass, config_entry):
         _LOGGER.error("Unable to communicate with Tesla API: %s", ex.message)
         return False
     _async_save_tokens(hass, config_entry, access_token, refresh_token)
+    coordinator = DataUpdateCoordinator(
+        hass,
+        _LOGGER,
+        # Name of the data. For logging purposes.
+        name="tesla",
+        update_method=async_update_data,
+        # Polling interval. Will only be polled if there are subscribers.
+        update_interval=timedelta(seconds=MIN_SCAN_INTERVAL),
+    )
+    # Fetch initial data so we have data when entities subscribe
     entry_data = hass.data[DOMAIN][config_entry.entry_id] = {
         "controller": controller,
+        "coordinator": coordinator,
         "devices": defaultdict(list),
         DATA_LISTENER: [config_entry.add_update_listener(update_listener)],
     }
-    _LOGGER.debug("Connected to the Tesla API")
+    _LOGGER.debug("Connected to the Tesla API.")
```

Comment:
i believe no periods are preferred for logging strings

Reply:
Thanks. I'll fix that.